packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,17 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map or as these directories:\n  ` +
+                searchQueue.join(`\n  `)
```

Comment:
Does this look good?

```js
throw new UnableToResolveError(
  fromModule,
  toModuleName,
  `Module does not exist in the module map or as these directories:
     ${searchQueue.join('\n  ')}

   This might be related to https://github.com/facebook/react-native/issues/4968
   To resolve try the following:
     1. Clear watchman watches: \`watchman watch-del-all\`.
     2. Delete the \`node_modules\` folder: \`rm -rf node_modules && npm install\`.
     3. Reset packager cache: \`rm -fr $TMPDIR/react-*\` or \`npm start -- --reset-cache\`.`
);
```

LMK if there are any changes needed